docs/reports/requirements/1_proj-desc.md

```diff
@@ -62,4 +62,38 @@ Note: the User section will be color-coded to show what's already in place and w
 
 ## PDF Report
 
-I will write my PDF report section here to later be incorporated into the full document.
+### High-level Description (about one paragraph)
+
+The objective for the Speaking Portal Project is to design, develop, and deploy a lip-sync animation (LSA) API for the
+Kukarella Table Reads text-to-speech (TTS) web application. This API will serve as an animation generating add-on for
+this system so that the user can both listen to and watch their avatar speak the user-provided text.
+
+### User groups
+
+In this project, there are two main user groups who will be using the software: general users and developers. The
+general users are the general public who use the Kukarella web application; this could be anyone who would like to
+listen to rather than read some text in any language with a customized avatar. The developers are the Kukarella
+employees who will be responsible for incorporating the API into their current system and adding more customizable
+assets for the avatars in the future.
+
+In the current Kukarella Table Reads TTS web application, the first step is for the user to create one or more actors.
+The user gives the actor a name, selects a voice from a list of voices they can preview, selects a language from the
+list of approximately 150 languages, and selects or uploads an image of the actor. After creating the actor and
+entering the text they want to listen to, the user presses the play button, and then waits for the application to
+process and generate the audio, which is then auto-played once the audio is complete.
+
+Once our API is incorporated into this system, if the user would like to both watch and listen to their avatar speak
+the provided text, the only difference in the user interface is that the user will select one of the "premium" avatar
+images for their actor. After pressing the play button, the user will wait for the Kukarella application and the API to
+process and generate a video of their avatar speaking the text with lips synchronized to the sounds produced in their
+generated speech.
+
+As for the developers, they will need to call the API within the Kukarella application. After receiving the user input
+and after the text is converted to speech in an audio file, the developer will call our API instead of providing the
+audio file as output for the user to listen to. When calling our API, the user-selected language, user-provided text,
```

Comment:
```suggestion
When calling our API, the user-selected language, user-provided text,
```
This change  will be required if changes are made in the above suggestion. Otherwise ignore.

Reply:
Removed, and will show next copy soon! Thanks for making it smoother!